src/app/COMPONENTS/create-task/create-task.component.ts

```diff
@@ -37,11 +39,12 @@ export class CreateTaskComponent {
   createTask() {
     if (this.createTaskForm?.valid) {
       const task: Task = this.createTaskForm.value
-      this.taskService.createNewTask(task)
+      this.taskService.createNewTask(task)?.pipe(take(1)).subscribe()
```

Comment:
instead of navigating after the method, you should do the navigation inside the subscribe